solved issue  #10 (added UI for 404 not found )

Kanavpreet-Singh opened the pull request:
![Screenshot 2025-02-14 114024](https://github.com/user-attachments/assets/cd35ebb7-87cd-45b3-8d68-58aa400d20b3)

krittika2004 commented:
Hey, 
That's a great try👏🎉. Process wise your PR was correct🙌✨. 
But, since, our website is related to flights so, try to introduce that theme a bit into this page. 
You can try to be a bit more creative.

Kanavpreet-Singh commented:
ok, ill add a bit more creativity .. pl assign the issue to me @krittika2004

Kanavpreet-Singh commented:
![Screenshot 2025-02-15 234854](https://github.com/user-attachments/assets/01bfe22f-1144-4957-bc93-a75858579ef8)
added animation and changed content of 404 error page @krittika2004

krittika2004 commented:
Hey, @Kanavpreet-Singh  there are some changes in our project repo from our side. You need to commit the changes again. Also, please refer only flight prediction final folder to refer and add files. Also, please attach screenshot of the page you made.
Thanks.

Kanavpreet-Singh commented:
![image](https://github.com/user-attachments/assets/20d797d5-db0f-4422-86d4-887cbef934c8)
@krittika2004 I have resolved conflicts with the main branch , and updated UI of 404 error page

krittika2004 commented:
Looks great! 
Merging your PR
Thanks for contributing
